.gitignore

```diff
@@ -0,0 +1,36 @@
+*.apk
+*.aar
+*.ap_
+*.aab
+*.dex
+*.class
+*.log
+*.iml
+*.hprof
+bin/
+gen/
+out/
+.gradle/
+build/
+proguard/
+.navigation/
+captures/
+.cxx/
+local.properties
+.DS_Store
+.externalNativeBuild
+.idea/workspace.xml
+.idea/tasks.xml
+.idea/gradle.xml
+.idea/assetWizardSettings.xml
+.idea/dictionaries
+.idea/libraries
+.idea/jarRepositories.xml
+.idea/caches
+.idea/modules.xml
+.idea/navEditor.xml
+.idea/gitignore
+.idea/compiler.xml
+.idea/deploymentTargetDropDown.xml
+vcs.xml
```

Comment:
@js-gonzalez I'd suggest you keep the comments to document this part.

Reply:
Agreed!